THIRD_PARTY_LICENSES

```diff
@@ -1,246 +1,232 @@
+third-party archives.
 
 Copyright [yyyy] [name of copyright owner]
 
 Licensed under the Apache License, Version 2.0 (the "License");
-
 you may not use this file except in compliance with the License.
-
 You may obtain a copy of the License at
 
 http://www.apache.org/licenses/LICENSE-2.0
 
 Unless required by applicable law or agreed to in writing, software
-
 distributed under the License is distributed on an "AS IS" BASIS,
-
 WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-
 See the License for the specific language governing permissions and
-
 limitations under the License.
 
-* For  Amazon.CDK see also this required NOTICE:
-    AWS Cloud Development Kit (AWS CDK)
-    Copyright 2018-2018 Amazon.com, Inc. or its affiliates. All Rights
-    Reserved.
-* For Amazon.CDK.CloudAssembly.Schema see also this required NOTICE:
-    AWS Cloud Development Kit (AWS CDK)
-    Copyright 2018-2018 Amazon.com, Inc. or its affiliates. All Rights
-    Reserved.
-* For Amazon.CDK.CXAPI see also this required NOTICE:
-    AWS Cloud Development Kit (AWS CDK)
-    Copyright 2018-2018 Amazon.com, Inc. or its affiliates. All Rights
-    Reserved.
-* For Amazon.CDK.RegionInfo see also this required NOTICE:
+* For Amazon.CDK.Lib see also this required NOTICE:
     AWS Cloud Development Kit (AWS CDK)
-    Copyright 2018-2018 Amazon.com, Inc. or its affiliates. All Rights
-    Reserved.
+    Copyright 2018-2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+* For AWSSDK.AppRunner see also this required NOTICE:
+    Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+* For AWSSDK.CloudFront see also this required NOTICE:
+    Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+* For AWSSDK.CloudWatchEvents see also this required NOTICE:
+    Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+* For AWSSDK.DynamoDBv2 see also this required NOTICE:
+    Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+* For AWSSDK.EC2 see also this required NOTICE:
+    Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+* For AWSSDK.ElasticLoadBalancingV2 see also this required NOTICE:
+    Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+* For AWSSDK.S3 see also this required NOTICE:
+    Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+* For AWSSDK.SimpleNotificationService see also this required NOTICE:
+    Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+* For AWSSDK.SQS see also this required NOTICE:
+    Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
 * For Amazon.JSII.Runtime see also this required NOTICE:
     AWS Cloud Development Kit (AWS CDK)
-    Copyright 2018-2018 Amazon.com, Inc. or its affiliates. All Rights
-    Reserved.
+    Copyright 2018-2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
Do we need the year for the copyright. If we do I'm sure 2018-2018 doesn't make sense.

Reply:
This was largely generated from the internal tooling based on what is in the LICENSE/NOTICE files in the dependent repos:
1. CDK did update to _"2018 - 2022"_ since we first took the dependency, updated in a new commit - https://github.com/aws/aws-cdk/blob/99e12b5e6520bdfb64724b409a8a0609f753ebc3/LICENSE#L189
2. JSII still only lists 2018, so prefer to match them - https://github.com/aws/jsii/blob/8da74aaa5df97571908cc34d91de9256b7e3fe48/LICENSE#L190